core/src/main/scala/kafka/server/BrokerLifecycleManager.scala

```diff
@@ -577,10 +563,9 @@ class BrokerLifecycleManager(
   }
 
   private def scheduleNextCommunicationAfterFailure(): Unit = {
-    val delayMs = resendExponentialBackoff.backoff(failedAttempts)
-    failedAttempts = failedAttempts + 1
     nextSchedulingShouldBeImmediate = false // never immediately reschedule after a failure
-    scheduleNextCommunication(NANOSECONDS.convert(delayMs, MILLISECONDS))
```

Comment:
Are there other retriable errors that we could see here that should retry sooner than the heartbeat?